include/filters/FromXMLFilter.h

```diff
@@ -27,6 +27,9 @@ class FromXMLFilter : public FilterBase
 
   FromXMLFilter(const InputParameters & parameters);
 
+  /// Override to return false as we don't know what each bin represents.
+  virtual bool skipBin(const unsigned int bin) const override { return false; }
```

Comment:
would it make more sense for the base class to define the implementation that results false, and then have only the necessary daughter classes override?